app/controllers/EventReportController.scala

```diff
@@ -45,84 +45,98 @@ class EventReportController @Inject()(
 
   def removeUserAnswers: Action[AnyContent] = Action.async {
     implicit request =>
-      withPstr { pstr => {
-        eventReportService.removeUserAnswers(pstr)
-        Future.successful(Ok)
-      }
+      withAuth.map { case Credentials(externalId, psaPspId) =>
+        eventReportService.removeUserAnswers(externalId)
+        Ok
       }
   }
 
   def saveUserAnswers: Action[AnyContent] = Action.async {
     implicit request =>
-      withPstrOptionEventTypeYearVersionAndBody {
-        case (pstr, Some(Tuple3(eventType, year, version)), userAnswersJson) =>
-          logger.debug(message = s"[Save Event: Incoming-Payload]$userAnswersJson")
-          EventType.getEventType(eventType) match {
-            case Some(et) =>
-              eventReportService.saveUserAnswers(pstr, et, year, version, userAnswersJson).map(_ => Ok)
-            case _ => Future.failed(new NotFoundException(s"Bad Request: eventType ($eventType) not found"))
-          }
-        case (pstr, None, userAnswersJson) =>
-          eventReportService.saveUserAnswers(pstr, userAnswersJson).map(_ => Ok)
+
+      withAuth.flatMap { case Credentials(externalId, psaPspId) =>
```

Comment:
These psaPspIds aren't used so change to _

Reply:
Done